### ISSUE TITLE
Manifest error

### ISSUE THREAD

**Issue by trustfall** [tipsi/tipsi-stripe#194]
Followed instructions for adding Android, I get the following error when building in Android Studio: 

```
Error:Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute data@scheme at AndroidManifest.xml requires a placeholder substitution but no value for <tipsiStripeRedirectScheme> is provided.
```

**Comment by trustfall** [comment]
Nevermind, further down in the docs

**Comment by isnifer** [comment]
I will reopen issue, we need to update docs

**Comment by adlondon** [comment]
Also having this issue for android. How did you solve it @trustfall

**Comment by paynd** [comment]
Just add 
```
manifestPlaceholders = [
            tipsiStripeRedirectScheme: "example"
        ]
```
into defaultConfig section of your app build.gradle
I'm going to update documentation asap with this asap.

**Comment by paynd** [comment]
@adlondon Actually we alredy had this in out [docs](https://github.com/tipsi/tipsi-stripe#android-2)

**Comment by kylealwyn** [comment]
No matter where I put `manifestPlaceholders` in build.grade, still getting this error. I've tried defaultConfig, buildTypes and productFlavors

Current config: 
```gradle
    defaultConfig {
        applicationId "co.myapp"
        minSdkVersion 19
        targetSdkVersion 26
        versionCode 6
        versionName "1.2.1"
        multiDexEnabled true
        
        manifestPlaceholders = [
                tipsiStripeRedirectScheme: "scheeeme"
        ]
        
        ndk {
            abiFilters "armeabi-v7a", "x86"
        }
    }
```

Current Output:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tipsi-stripe:processDebugAndroidTestManifest'.
> Manifest merger failed : Attribute data@scheme at manifestMerger2118535579845422616.xml requires a placeholder substitution but no value for <tipsiStripeRedirectScheme> is provided.
```

Android Studio 3.0.1
Gradle 4.3

**Comment by trustfall** [comment]
I found for me it doesn't recognize the redirect scheme at all. If I use an ejected build of CRNA, it will build with the expokit scheme given, but only if I build directly to device or emulator. If I just rebuild, it will give the same error. Expokit is kind of a different animal, but maybe there's something there.

**Comment by kylealwyn** [comment]
Interesting, thanks for the followup. Not using CRNA or Expo here though. Downgraded to 4.1.2 in the meantime til I or someone smarter than me can figure it out

**Comment by trustfall** [comment]
worst case scenario, you can always hardcode the value in the tipsi manifest file.

**Comment by BearJS** [comment]
Exact similar issues @kylealwyn I have added the manifest placeholder however my stack trace shows the following issue:

> Task :app:processDebugResources
Failed to execute aapt
com.android.ide.common.process.ProcessException: Failed to execute aapt

**Comment by isnifer** [comment]
@BearJS https://github.com/tipsi/tipsi-stripe/blob/master/example/android/gradle.properties#L21

**Comment by BearJS** [comment]
@isnifer the value for these properties are already correct

**Comment by adlondon** [comment]
Not sure if it helps anyone here, but I was still having this issue running an outdated version of NPM. Don't know how it got that way, but updating to NPM @5.5.1 and Node @8.9.3 set me right

**Comment by paynd** [comment]
@BearJS can you post buildToolsVersion, compileSdkVersion, defaultConfig { ... } and dependencies { ... } sections from your application level build.gradle?
@trustfall @kylealwyn can you try #205 ?

**Comment by kingarawana** [comment]
I'm having the same issue. 

`Execution failed for task ':tipsi-stripe:processDebugAndroidTestManifest'.
> Manifest merger failed : Attribute data@scheme at manifestMerger1426022766957738802.xml requires a placeholder substitution but no value for <tipsiStripeRedirectScheme> is provided.`

I tried doing as the docs suggested and added it under android.defaultConfig.manifestPlaceholders but it failed. I also tried adding it under buildTypes.debug.defaultConfig.manifestPlaceholders since it said "processDebug.." but that also didn't help.

Here are my current project configs

`applicationId "com.myapp"
        minSdkVersion 17
        targetSdkVersion 22
        versionCode 9
        versionName "1.5.4"
        multiDexEnabled true`

I also have several several product flavors. Not sure if that would have anything to do with it.
